### DISCUSSION TITLE
StyleSheet starts accumulating styles after upgrading from 0.14 to 0.19

### DISCUSSION THREAD

**Discussion by zhiduyan** [necolas/react-native-web#2534] [Help]
Hi there,

After upgrading RNW from 0.14 to 0.19, I noticed that `AppRegistery/renderApplication` has a change in `getStyleElement()` and now it accumulates styles across SSR requests. How can I get it back to the original behavior, where fresh StyleSheet is returned for each SSR?

Here is what it looks like:
```
import ReactDOMServer from "react-dom/server";
import { AppRegistry, StyleSheet } from "react-native-web";

...

// an ad hoc style
const myStyle = StyleSheet.create({...});
ReactDOMServer.renderToString(React.createElement(View, { myStyle }));

...

// call getStyleElement to reset style
const { getStyleElement } = AppRegistry.getApplication(MyApplicationName, {});
getStyleElement();

// expect to get fresh style
const { getStyleElement } = AppRegistry.getApplication(MyApplicationName, {});
const styleElement = getStyleElement();
const styleTag = ReactDOMServer.renderToStaticMarkup(styleElement);
```

Thanks in advance!
Jack

**Comment by necolas** [comment]
https://github.com/necolas/react-native-web/releases/tag/0.18.0

**Reply by zhiduyan** [reply]
Wow! Thanks for the prompt reply! I'll read the doc and let you know how it works out.

**Reply by zhiduyan** [reply]
I'm sorry but I'm new to these and I still don't know how to fix the issue after reading the release note. Would you mind to give me a little more specifics?

**Reply by zhiduyan** [reply]
Or could you tell me more about the motivation for this change? I don't think we should accumulate styles across requests. This could cause unnecessary styles to be included in later requests.

**Reply by zhiduyan** [reply]
https://github.com/necolas/react-native-web/issues/778
